### DISCUSSION TITLE
Engine.IO v4 Release - feedback

### DISCUSSION THREAD

**Discussion by darrachequesne** [socketio/socket.io#3644] [General]
Related blog post: https://socket.io/blog/engine-io-4-release/

If you have any remarks/suggestions about this release, let's discuss about it here!

**Comment by RosenTomov** [comment]
Great news and thank you for maintaining the project!

**Comment by d2e1** [comment]
First of all thanks for the release. I have started using  for ping/pong change with socket io npm 2.3.0 by changing  the package.json of socket io node module. I know this is  not  official . First tests  did  not  show any error . I would like to ask that would there be a problem by using current  socketio  with engine io  4.0 untill official socket io release supports engineio 4.0?

**Reply by darrachequesne** [reply]
No, as long as you use the same Engine.IO version for both the server (`engine.io@4.0.0`) and the client (`engine.io-client@4.0.0`), it should work.
